src/app/views/authentication/profile/ProfileV9.tsx

```diff
@@ -147,18 +147,23 @@ const SignedInButton = forwardRef<
   React.ComponentProps<typeof Button>
 >((props, ref) => {
   const styles = useHeaderStyles();
+  const profile = useAppSelector((state) => state.profile);
+  const user = profile.user;
```

Comment:
```suggestion
  const user = useAppSelector((state) => state.profile.user);
```
Just a nit. This is all good to go.